concepts/numbers/about.md

```diff
@@ -55,6 +55,73 @@ There are three types of maximum (and minimum / maximum negative) values for num
+- `POSITIVE_INFINITY`, divided by any negative value except `NEGATIVE_INFINITY`, is `NEGATIVE_INFINITY`.
+- Zero multiplied by `POSITIVE_INFINITY` is `NaN`.
+
+[`negative-infinity`][negative-infinity] is the same as `POSITIVE_INFINITY` but with a negative sign :
+- `NEGATIVE_INFINITY`, divided by either `NEGATIVE_INFINITY` or `POSITIVE_INFINITY`, is `NaN`.
+- `NEGATIVE_INFINITY`, divided by any negative value except `NEGATIVE_INFINITY`, is `POSITIVE_INFINITY`.
+- Any positive value, including `POSITIVE_INFINITY`, multiplied by `NEGATIVE_INFINITY` is `NEGATIVE_INFINITY`.
+
+
+## NaN or Not-a-Number
+
+The [`NaN`][NaN] is a variable in global scope. There are five different types of operations returning `NaN`:
+
+-  Number cannot be parsed (e.g. parseInt("blabla") or Number(undefined))
+-  Math operation where the result is not a real number (e.g. Math.sqrt(-1))
+-  Operand of an argument is NaN (e.g. 7 ** NaN)
+-  Indeterminate form (e.g. 0 * Infinity, or undefined + undefined)
+-  Any operation that involves a string and is not an addition operation (e.g. "foo" / 3)
+
+## Exponentiation 
+
+You can put your [`exponentiation`][exponentiation] by using `**`.
+Here are some examples of exponentiation:
+```javascript
+2 ** 3   // 8
+3 ** 2   // 9
+3 ** 2.5 // 15.588457268119896
+10 ** -1 // 0.1
+NaN ** 2 // NaN
+```
+The exponentiation operator is right-associative: `a ** b ** c` is equal to `a ** (b ** c)`.
+Beware that JavaScript is different than most languages with exponentiation, it is impossible to write an ambiguous exponentiation expression. That is, you cannot put a unary operator `(+/-/~/!/delete/void/typeof)` immediately before the base number, doing so will cause a `SyntaxError`.
+In other words this : `-2 ** 2` will cause an error, but this is correct `-(2**2)`.
+Also do not use `^` like in every other language, as it is reserved for `bitwise logical XOR operator`!
+
+## Underscore in numbers
+
+An [`underscore`][underscore] can be used in numbers to help readability. However you can place underscores only between digits, you cannot place underscores in the following places:
+- At the beginning or end of a number
+- Adjacent to a decimal point in a floating point literal
+- Prior to an F or L suffix 
+- In positions where a string of digits is expected
+
+Here are some examples on what you can and can not do :
+```javascript
+int x4 = 5_______2;        // OK (decimal literal)
+int x2 = 5_2;              // OK (decimal literal)
+float pi1 = 3_.1415F;      // Invalid; cannot put underscores adjacent to a decimal point
+int x11 = 052_;            // Invalid; cannot put underscores at the end of a number
+int x5 = 0_x52;            // Invalid; cannot put underscores in the 0x radix prefix
+int x6 = 0x_52;            // Invalid; cannot put underscores at the beginning of a number
+long socialSecurityNumber1 = 999_99_9999_L;         // Invalid; cannot put underscores prior to an L suffix
+```
```

Comment:
This doesn't look like JavaScript!

Reply:
You're very right indeed, sorry for the mixup I'll fix that